lib/Textmaster/Api/Locale.php

```diff
@@ -62,4 +65,34 @@ public function countries($locale)
     {
         return $this->get('public/countries/'.rawurlencode($locale));
     }
+
+    /**
+     * List all available locales for an ability.
+     *
+     * @param string      $ability Can be one of ProjectInterface::ACTIVITY
+     * @param int         $page
+     * @param string|null $locale
+     *
+     * @return array
+     */
+    public function abilities($ability, $page, $locale = null)
+    {
+        $abilities = ProjectModel::getAllowedActivities();
```

Comment:
It could make sense to move them in the API layer as this values are defined by the TextMaster API.
Do you want me to make the change in this PR ?